lib/utils.ts

```diff
@@ -24,7 +24,7 @@ export function getWordAtPosition(editor: TextEditor, position: Point): Range {
 
 export function escapeRegExp(string: string): string {
   // From atom/underscore-plus.
-  return string.replace(/[-/\\^$*+?.()|[\]{}]/g, "\\$&")
+  return string.replace(/[$()*+./?[\\\]^{|}-]/g, "\\$&")
```

Comment:
This change seems correct, but we can discuss it more.

Reply:
It is the same but odd that prettier changes it.

Reply:
It is not prettier, it is the optimize-regex plugin for Eslint. I recently added it to the Eslint config